### PR TITLE
lnrpc: adds active_only flag to listchannels command

### PR DESCRIPTION
**Pull request by stridentbean**
This commit implements an active_only flag for the listchannels command. By default, listchannels list all channels active or not active. Adding this flag will result in listchannels only returning active channels. The absence of this flag will result in listing all channels.

### CODE REVIEW

**Comment by stridentbean** [comment]
I still have to add tests. Does anyone know what the formatting issue is about in commands.go? I'm using atom as an editor.

File: lnrpc/rpc.pb.go [hunk 1]
```diff
@@ -999,13 +999,22 @@ func (m *ActiveChannel) GetCsvDelay() uint32 {
 }
 
 type ListChannelsRequest struct {
+	// / Whether this channel should be private, not announced to the greater network.
```
Comment (stridentbean) [inline lnrpc/rpc.pb.go]: woops gotta fix this

**Comment by Roasbeef** [comment]
What's the direct value in this? Why not just filter out by active if you're doing this programmatically?

**Comment by stridentbean** [comment]
@Roasbeef Could be done either way. I saw that there was a flag for it already that wasn't implemented and assumed someone wanted it done at some point. I'm fine either way. Just trying to get a handle on the code base finally.

**Pull request closed without merging**